src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -5853,81 +5859,129 @@ subroutine atm_init_coupled_diagnostics(state, time_lev, diag, mesh, configs, &
       call mpas_pool_get_array(mesh, 'zb_cell', zb_cell)
       call mpas_pool_get_array(mesh, 'zb3_cell', zb3_cell)
 
+      MPAS_ACC_TIMER_START('atm_init_coupled_diagnostics [ACC_data_xfer]')
+      ! copyin invariant fields
+      !$acc enter data copyin(cellsOnEdge,nEdgesOnCell,edgesOnCell, &
+      !$acc                   edgesOnCell_sign,zz,fzm,fzp,zb,zb3, &
+      !$acc                   zb_cell,zb3_cell)
+
+      ! copyin the data that is only on the right-hand side
+      !$acc enter data copyin(scalars(index_qv,:,:),u,w,rho,theta, &
+      !$acc                           rho_base,theta_base)
+
+      ! copyin the data that will be modified in this routine
+      !$acc enter data copyin(theta_m,rho_zz,ru,rw,rho_p,rtheta_base, &
```

Comment:
Since the goal of this routine is to compute each of these fields, I think we only need to `create` them and not `copyin`.

Reply:
Agreed, addressed in fixup commit [0ee0d43](https://github.com/MPAS-Dev/MPAS-Model/pull/1216/commits/0ee0d433ab53391b272be60f5afd699ea5e0c753)